x-pack/plugin/transform/qa/single-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformRestTestCase.java

```diff
@@ -88,7 +87,7 @@ protected void createReviewsIndex(
                 min = 10 + (i % 49);
             }
             int sec = 10 + (i % 49);
-            String location = (user + 10) + "," + (user + 15);
+            String location = (((user + 10) % 180) - 90) + "," + (((user + 15) % 360) - 180);
```

Comment:
Bulk upload was failing for the test `TransformPivotRestSpecialCasesIT::testRestrictiveBucketSelector` because the lat,lon values were outside of the range.